ops/terraform/services/eft/modules/bfd_eft_outbound_o11y/lambda_src/outbound_slack_notifier/outbound_slack_notifier.py

```diff
@@ -107,6 +107,9 @@ def handler(event: dict[Any, Any], context: LambdaContext):  # pylint: disable=u
             status_notification = StatusNotification(**raw_notification)
 
             logger.append_keys(notification_type=status_notification.type.value)
+         
+            if (status_notification.type == NotificationType.TRANSFER_SUCCESS):
```

Comment:
Also exclude `FILE_DISCOVERED` since we would like for those messages to not send to Slack as well.